spec/namespaces/_core.yaml

```diff
@@ -2523,9 +2523,6 @@ components:
                 $ref: '../schemas/_common.yaml#/components/schemas/Duration'
               scroll_id:
                 $ref: '../schemas/_common.yaml#/components/schemas/ScrollId'
-            required:
```

Comment:
Why need to change this file? Seems not related with ML APIs

Reply:
This is part of a [different commit](https://github.com/opensearch-project/opensearch-api-specification/pull/732/files). I experienced a merge conflict before and resolving it caused some of the commits to be added to this PR.